### ISSUE TITLE
Formatting change.

### ISSUE THREAD

**Issue by Devieth** [LaxSlash/Halo-CE-Discord-Reporter#10]
![idea](https://cloud.githubusercontent.com/assets/5672855/19957209/92b279aa-a162-11e6-9ac5-fac9618b685c.jpg)

**Comment by LaxSlash** [comment]
Line spacing might be tricky. Not completely sure that it's possible, since everything's being sent in fields in the embed object.

Having said that, I _could_ theoretically just make it all one big Description block, I think...

**Comment by LaxSlash** [comment]
Formatting changed successfully. May have to wait on a Discord fix for why the CD Hash is going to a new line, but this can be marked as closed and that put into a new issue.
